### PR TITLE
Notes from Design System Day 2023

### PR DESCRIPTION
**Pull request by frankieroberto**
Notes from the conference and the talk I gave about contributing to design systems.

### CODE REVIEW

File: app/posts/design-system-day-2023.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+This month I was thrilled to take part in [Design System Day 2023](https://design-system.service.gov.uk/community/design-system-day-2023/), a 2-day conference run by the GOV.UK Design System team to collaborate and share knowledge about design systems.
+
+Here are my notes from the conference, as well as an outline of the talk I gave.
+
+## Day 1
+
+The first day was held in-person at Our Dynamic Earth in Edinburgh, and included sessions on research operations, the [Scottish Government Digital Design System](https://designsystem.gov.scot) and trauma-informed research. I also took part in a panel discussion on how design systems manage contributions, where we discussed how and why to encourage teams using design systems to contribute back.
+
+The final session of the day was a keynote by Tash Willcocks called ‘Design systems don’t solve problems, people do’ (love the Goldie Lookin Chain reference!) – where Tash implored design system teams to continually "socialise" their design systems by being in the spaces where teams using the design hang out, and making the effort to understand the reasons why a design might not be being adopted.
```
Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:18]: Do you have a link for Task Willcocks’ website?

```suggestion
The final session of the day was a keynote by Tash Willcocks called ‘Design systems don’t solve problems, people do’ (love the Goldie Lookin Chain reference!) – where Tash implored design system teams to continually "socialise" their design systems by being in the spaces where teams using the design hang out, and making the effort to understand the reasons why a design might not be being adopted.
```

Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:18]: Brackets of en-dash, pick one!

```suggestion
The final session of the day was a keynote by Tash Willcocks called ‘Design systems don’t solve problems, people do’ – love the Goldie Lookin Chain reference! – where Tash implored design system teams to continually "socialise" their design systems by being in the spaces where teams using the design hang out, and making the effort to understand the reasons why a design might not be being adopted.
```

Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:18]: 💅

```suggestion
The final session of the day was a keynote by Tash Willcocks called ‘Design systems don’t solve problems, people do’ (love the Goldie Lookin Chain reference!) – where Tash implored design system teams to continually “socialise” their design systems by being in the spaces where teams using the design hang out, and making the effort to understand the reasons why a design might not be being adopted.
```

File: app/posts/design-system-day-2023.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+The videos from each of these sessions will be released online soon.
+
+## Day 2
+
+The second day was held as a multi-track online conference.
+
+I enjoyed attending a fireside chat with [Ethan Marcotte](https://ethanmarcotte.com), which touched upon the history of responsive design, as well as the many different forms of design systems and brand guidelines. 
+
+In the middle of the day, Vicky Teinaki gave a fantastic keynote titled [Design pattern histories](https://www.vickyteinaki.com/blog/design-pattern-histories/), where she delved deep into the history of now-familiar design patterns like pull-to-refresh, news feeds and task lists, as well ideas now discouraged like [external link icons](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/?ref=vickyteinaki.com) and [carousels](https://shouldiuseacarousel.com).
```
Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:28]: Is the `ref` query needed?

```suggestion
In the middle of the day, Vicky Teinaki gave a fantastic keynote titled [Design pattern histories](https://www.vickyteinaki.com/blog/design-pattern-histories/), where she delved deep into the history of now-familiar design patterns like pull-to-refresh, news feeds and task lists, as well ideas now discouraged like [external link icons](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/) and [carousels](https://shouldiuseacarousel.com).
```

File: app/posts/design-system-day-2023.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+Her conclusion was that it pays to understand the history of where a pattern came from, to keep testing and sharing your findings, and to remember that ‘best practice’ can and will continue to change!
```
Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:30]:
```suggestion
Her conclusion was that it pays to understand the history of where a pattern came from, to keep testing and sharing your findings, and to remember that best practice can and will continue to change!
```

File: app/posts/design-system-day-2023.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+Both of these were entirely new patterns or component, but contributions come in lots of different sizes, and so I also talked through 2 smaller contributions.
```
Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:46]:
```suggestion
Both of these were entirely new patterns or components, but contributions come in lots of different sizes, and so I also talked through 2 smaller contributions.
```

File: app/posts/design-system-day-2023.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+The second small contribution was some new guidance on the [Date input](https://design-system.service.gov.uk/components/date-input/) to suggest that teams should accept dates where the month is written in words ('aug' or 'august') as well as as a number. This was prompted by data from my service showing common validation errors, which is a great source of research.
+
+As well as talking about my own contributions, I also wanted to give a shout out to others who have contributed to the wider government Design System ecosystem, including here on X-GOVUK.
+
+Paul Lloyd has led 2 super useful plugins for the Prototype Kit. The first, [Prototype filters](https://x-govuk.github.io/govuk-prototype-filters/), is really useful if you are prototyping a service like a case management system which has a lot of data in it. The second, [Prototype components](https://x-govuk.github.io/govuk-prototype-components/), contains a selection of components which are widely used across services, but haven’t yet made their way into the GOV.UK Design System.
```
Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:56]: ☺️

File: app/posts/design-system-day-2023.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+Peter Yates has led [GOV.UK Components for Ruby on Rails](https://govuk-components.netlify.app) and [Form Builder for Ruby on Rails](https://govuk-form-builder.netlify.app), 2 ruby libraries which make it far easier for teams to implement the GOV.UK Design System within the Ruby on Rails framework. These are now depended upon by dozens of services across government departments, including by the new [GOV.UK Forms service](https://www.forms.service.gov.uk) being developed by GDS itself. This is a great example of a cross-government collaboration saving teams from duplicating effort, and I suggest that in future we could look to better integrate this implementation with the Design System itself, as a third option alongside HTML and Nunjucks.
```
Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:58]:
```suggestion
Peter Yates has led [GOV.UK Components for Ruby on Rails](https://govuk-components.netlify.app) and [Form Builder for Ruby on Rails](https://govuk-form-builder.netlify.app), 2 Ruby libraries which make it far easier for teams to implement the GOV.UK Design System within the Ruby on Rails framework. These are now depended upon by dozens of services across government departments, including by the new [GOV.UK Forms service](https://www.forms.service.gov.uk) being developed by GDS itself. This is a great example of a cross-government collaboration saving teams from duplicating effort, and I suggest that in future we could look to better integrate this implementation with the Design System itself, as a third option alongside HTML and Nunjucks.
```

Comment (paulrobertlloyd) [inline app/posts/design-system-day-2023.md:58]: Did you suggest this in your talk?

```suggestion
Peter Yates has led [GOV.UK Components for Ruby on Rails](https://govuk-components.netlify.app) and [Form Builder for Ruby on Rails](https://govuk-form-builder.netlify.app), 2 ruby libraries which make it far easier for teams to implement the GOV.UK Design System within the Ruby on Rails framework. These are now depended upon by dozens of services across government departments, including by the new [GOV.UK Forms service](https://www.forms.service.gov.uk) being developed by GDS itself. This is a great example of a cross-government collaboration saving teams from duplicating effort, and I suggested that in future we could look to better integrate this implementation with the Design System itself, as a third option alongside HTML and Nunjucks.
```

**Review by paulrobertlloyd — COMMENTED** [review]
Love it! Few typos/suggests. Also, does the last section need a few more headings to break it up a bit?

File: app/posts/design-system-day-2023.md (hunk 1 above)
Comment (frankieroberto) [reply app/posts/design-system-day-2023.md]: She doesn’t seem to have one...

**Pull request merged**